layouts/partials/navbar.html

```diff
@@ -1,23 +1,18 @@
 {{ $home     := site.BaseURL }}
 {{ $here     := .RelPermalink }}
 {{ $title    := site.Title }}
-{{ $logo     := site.Params.logos.navbar }}
+{{ $logo := printf "img/logos/%s" site.Params.logos.navbar | relURL }}
 {{ $menu     := site.Menus.main }}
 {{ $versions := site.Params.versions }}
 {{ $latest   := index $versions 0 }}
 {{ $isDocs   := eq .Section "docs" }}
 {{ $version  := index (split .File.Path "/") 1 }}
 {{ $isLatest := eq $version $latest }}
-<nav class="navbar is-black is-fixed-top" role="navigation" aria-label="main navigation">
+<nav class="navbar is-white is-fixed-top" role="navigation" aria-label="main navigation">
```

Comment:
White is the default, so you don't need to specify that

Reply:
Removing!